scripts/ci-metrics-publisher.sh

```diff
@@ -6,13 +6,23 @@ data=""
 
 for ((i = 1; i <= $#; i++ )); do
   remainder="${!i}"
-  first="${remainder%%=*}"; remainder="${remainder#*=}"
+  # Find everything until last = character (= is included in the result)
+  # This allows to add tags to metric names
+  metricName=$(grep -o "\(.*\)=" <<< $remainder)
+  # Get the metric value
+  value=${remainder#"$metricName"}
+  # Remove remaining = character from metric name
+  metricName=${metricName%?};
+
+
   if [ -n "$data" ]; then
     data="$data,"
   fi
-  data=''$data'{"name": "'${first}'", "value": '${remainder}', "interval": 60, "mtype": "gauge", "time": '$(date +%s)'}'
+  data=''$data'{"name": "'${metricName}'", "value": '${value}', "interval": 60, "time": '$(date +%s)'}'
```

Comment:
why is  "mtype": "gauge" removed? that makes the rollup correct (I think)

Reply:
ah, playgin with graphite. fixing